support-frontend/assets/pages/paper-subscription-checkout/helpers/__tests__/deliveryDays.js

```diff
@@ -26,15 +29,34 @@ describe('deliveryDays', () => {
     });
   });
 
-  // TODO: re-enable these once home delivery available again
-  xdescribe('getHomeDeliveryDays', () => {
+  describe('getHomeDeliveryDays', () => {
+    it('can work out the number of days till the next delivery', () => {
+      expect(daysTillNextDelivery(DeliveryDays.Monday, DeliveryDays.Tuesday)).toEqual(1);
+      expect(daysTillNextDelivery(DeliveryDays.Monday, DeliveryDays.Friday)).toEqual(4);
+      expect(daysTillNextDelivery(DeliveryDays.Friday, DeliveryDays.Monday)).toEqual(3);
+      expect(daysTillNextDelivery(DeliveryDays.Wednesday, DeliveryDays.Sunday)).toEqual(4);
+    });
+    it('can work out whether we can deliver on the next delivery date', () => {
+      expect(canDeliverOnNextDeliveryDay(DeliveryDays.Tuesday, DeliveryDays.Sunday, 4)).toEqual(true);
+      expect(canDeliverOnNextDeliveryDay(DeliveryDays.Wednesday, DeliveryDays.Saturday, 4)).toEqual(false);
+      expect(canDeliverOnNextDeliveryDay(DeliveryDays.Saturday, DeliveryDays.Monday, 3)).toEqual(false);
+      expect(canDeliverOnNextDeliveryDay(DeliveryDays.Wednesday, DeliveryDays.Sunday, 4)).toEqual(true);
+    });
     it('delivers the Sunday paper on the next Sunday', () => {
       const days = getHomeDeliveryDays(wednesday, 'Sunday');
       expect(formatMachineDate(days[0])).toEqual('2019-03-03');
     });
-    it('delivers the Monday paper on the next Monday even if it\'s a Sunday', () => {
-      const days = getHomeDeliveryDays(sunday, 'Everyday');
-      expect(formatMachineDate(days[0])).toEqual('2019-03-04');
+    it('delivers first Sixday paper 6 days later if ordered on a Thursday', () => {
+      const days = getHomeDeliveryDays(thursday, 'Sixday');
+      expect(formatMachineDate(days[0])).toEqual('2019-03-06');
+    });
+    it('delivers first Weekend paper 9 days later if ordered on a Thursday', () => {
+      const days = getHomeDeliveryDays(thursday, 'Weekend');
+      expect(formatMachineDate(days[0])).toEqual('2019-03-09');
+    });
+    it('delivers first Sunday paper 10 days later if ordered on a Thursday', () => {
```

Comment:
I would've put a few more cases in so we're covering more than thursday acquisitions?